### PR TITLE
Add Changelog format to PR template

### PR DESCRIPTION
**Pull request by kstiehl**
### Acceptance tests
- [ ] Have you added an acceptance test for the functionality being added?
- [ ] Have you run the acceptance tests on this branch?

### Release Note
Release note for [CHANGELOG](https://github.com/anexia-it/terraform-provider-anxcloud/blob/master/CHANGELOG.md):

```release-note
NONE
```

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### CODE REVIEW

File: .github/pull_request_template.md
Comment (LittleFox94) [inline .github/pull_request_template.md:26]: maybe add an example here, too, as you did in anexia-it/go-anxcloud#89?

**Review by LittleFox94 — CHANGES_REQUESTED** [review]
The template fun is not for this PR I think?

File: .github/pull_request_template.md
Comment (kstiehl) [reply .github/pull_request_template.md:26]: True

**Comment by LittleFox94** [comment]
On hold until template lookup from #71 is merged and only the changelog template is changed in this PR

**Comment by kstiehl** [comment]
Waiting for #73  to be merged. So the change of the template Identifier can be removed here

**Review by LittleFox94 — APPROVED** [review]

**Pull request merged**
